### PR TITLE
Designer fixes

### PR DESCRIPTION
**Pull request by shucon**
### Description

Branch from: https://github.com/phpmyadmin/phpmyadmin/pull/15586

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by williamdes** [comment]
Would you prefer that I rebase onto master before you start working ?

**Comment by shucon** [comment]
> Would you prefer that I rebase onto master before you start working ?

Sure, thanks :)

File: js/designer/page.js [hunk 1]
```diff
@@ -111,15 +111,15 @@ DesignerPage.loadFirstPage = function (db, callback) {
 };
 
 DesignerPage.showNewPageTables = function (check) {
-    var allTables = $('#id_scroll_tab').find('td input:checkbox');
+    var allTables = $('.scroll_tab_checkbox:checkbox');
     allTables.prop('checked', check);
     for (var tab = 0; tab < allTables.length; tab++) {
         var input = allTables[tab];
         if (input.value) {
-            var element = document.getElementById(input.value);
```
Comment (shucon) [inline js/designer/page.js:119]: @williamdes why have you appended `designer_table_` here? It's breaking the code.

File: js/designer/page.js [hunk 2]
```diff
@@ -111,15 +111,15 @@ DesignerPage.loadFirstPage = function (db, callback) {
...
+            var element = document.getElementById('designer_table_' + input.value);
```
Comment (shucon) [inline js/designer/page.js:119]: @williamdes why have you appended designer_table_ here? It's breaking the code.

**Comment by williamdes** [comment]
Please hard reset onto f6b373cd4871b79758e5afc2e3f329053c7858e2 from my repo :)

File: js/designer/page.js (hunk 2 above)
Comment (williamdes) [reply js/designer/page.js:119]: I did add `designer_table_` to have a consistent way to track the occurrences that use the node.
If you do not append `designer_table_` you will have no way to know what part of the code uses your node, etc..

Tracking is the key ;)

**Pull request closed without merging**